makefile

```diff
@@ -1,4 +1,4 @@
-.PHONY: test run-generators update-docs
+xr.PHONY: test run-generators update-docs
```

Comment:
What does this do and what does it have to do with the PR?

Reply:
Hmm, bit flip